ui/css/design-system/attributes.scss

```
@@ -50,6 +50,7 @@ $font-style: normal, italic, oblique;
 $font-size: 10px, 12px;
 $border-radius: (
   xs: 2px,
+  sm: 4px,
   md: 4px,
```

Update medium to 6 pixels
```suggestion
  md: 6px,
```